src/GdbConnection.cc

```diff
@@ -679,14 +679,16 @@ bool GdbConnection::query(char* payload) {
                  ";qXfer:libraries:read+"
                  ";qXfer:features:read+"
                  ";qXfer:auxv:read+"
-                 ";qXfer:exec-file:read+"
                  ";qXfer:siginfo:read+"
                  ";qXfer:siginfo:write+"
                  ";multiprocess+"
                  ";hwbreak+"
                  ";swbreak+"
                  ";ConditionalBreakpoints+"
                  ";vContSupported+";
+    if (!features().target_wine) {
+      supported << ";qXfer:exec-file:read+";
+    }
```

Comment:
Why are we doing this?

Reply:
That is also like winedbg handles it and while developing this I thought this is needed to work.
But I tested again with the current state and I can present this feature to gdb and it still works. I will remove this.